api-gateway/fcgi/asr/cpp/fcgi_asr.cpp

```diff
@@ -266,21 +266,21 @@ std::string asr(std::string params_str, std::string wave_filename) {
 
         CCAI_NOTICE("local inference");
         ie_result += "{\r\n";
```

Comment:
CRLF("\r\n") is a delimiter to separate the headers in HTTP protocol. It should not appear in HTTP body as a line separator in Linux/Unix environment, especially mix use with LF.
Ref: https://datatracker.ietf.org/doc/html/rfc7230#section-3